modules/core/shared/src/main/scala/lucuma/core/model/ObjectTracking.scala

```diff
@@ -37,6 +37,11 @@ object ObjectTracking:
       trackings.centerOfAt(i).map(CoordinatesAtVizTime(_))
     def baseCoordinates: Coordinates                 = trackings.centerOf
 
+  case class ConstantTracking(coordinates: Coordinates) extends ObjectTracking derives Eq:
```

Comment:
Seems good, I thought we could do the same  with a constant `SiderealTracking` but this may be more generic